modin/pandas/base.py

```diff
@@ -3665,6 +3666,95 @@ def __getitem__(self, key):
         else:
             return self._getitem(key)
 
+    def xs(
+        self,
+        key,
+        axis=0,
+        level=None,
+        drop_level: bool = True,
+    ):  # noqa: PR01, RT01, D200
+        """
+        Return cross-section from the Series/DataFrame.
+        """
+        axis = self._get_axis_number(axis)
+        labels = self.axes[axis]
+
+        if isinstance(key, list):
+            # deprecated in pandas, to be removed in 2.0
+            warnings.warn(
+                "Passing lists as key for xs is deprecated and will be removed in a "
+                + "future version. Pass key as a tuple instead.",
+                FutureWarning,
+            )
+
+        if level is not None:
+            if not isinstance(labels, pandas.MultiIndex):
+                raise TypeError("Index must be a MultiIndex")
+            loc, new_ax = labels.get_loc_level(key, level=level, drop_level=drop_level)
+
+            # create the tuple of the indexer
+            _indexer = [slice(None)] * self.ndim
+            _indexer[axis] = loc
+            indexer = tuple(_indexer)
+
+            result = self.iloc[indexer]
+            setattr(result, self._pandas_class._get_axis_name(axis), new_ax)
+            return result
+
+        if axis == 1:
+            if drop_level:
+                return self[key]
+            index = self.columns
+        else:
+            index = self.index
+
+        new_index = None
+        if isinstance(index, pandas.MultiIndex):
```

Comment:
```suggestion
        if axis == 1 and drop_level:
            return self[key]

        new_index = None
        if isinstance(labels, pandas.MultiIndex):
```
and further replace `index` with `labels`

Reply:
@dchigarev this is how it is done in pandas I guess, let's not deviate much from their version of the code.